sonic-slave-buster/Dockerfile.j2

```diff
@@ -342,8 +340,7 @@ RUN pip install \
 
 # For sonic config engine testing
 RUN pip install pyangbind==0.6.0
-# Note: force upgrade debian packaged jinja2, if installed
-RUN pip install --force-reinstall --upgrade "jinja2>=2.10"
+RUN pip install "jinja2>=2.10"
```

Comment:
can you pin down the version, for example jinja==2.11

Reply:
Done